plugins/modules/lightsail_snapshot.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: lightsail_snapshot
+version_added: "1.0.0"
+short_description: Creates snapshots of AWS Lightsail instances
+description:
+    - Creates snapshots of AWS Lightsail instances
+author:
+    - "Nuno Saavedra (@Nfsaavedra)"
+options:
+  state:
+    description:
+      - Indicate desired state of the target.
+    default: present
+    choices: ['present', 'absent']
+    type: str
+  region:
+    description: AWS region in which to create the instance snapshot.
+    required: true
+    type: str
+  snapshot_name:
+    description: Name of the new instance snapshot
+    required: true
+    type: str
+  instance_name:
+    description:
+      - Name of the instance to create the snapshot
+      - Required when I(state=present)
+    type: str
+  wait:
+    description:
+      - Wait for the instance snapshot to be created before returning.
+    type: bool
+    default: true
+  wait_timeout:
+    description:
+      - How long before I(wait) gives up, in seconds.
+    default: 300
+    type: int
+
+extends_documentation_fragment:
+- amazon.aws.aws
+- amazon.aws.boto3
+
```

Comment:
```suggestion
- amazon.aws.common.modules
- amazon.aws.region.modules
- amazon.aws.boto3
```

The fragments got renamed